src/lein_monolith/dependency.clj

```diff
@@ -176,8 +244,20 @@
      ; should appear *later* in the sequence.
      (let [roots (apply set/difference (set (keys m)) (map set (vals m)))]
        (when (empty? roots)
-         (throw (ex-info "Cannot sort the keys in the given map, cycle detected!"
-                         {:input m})))
+         (let [sort-cycles #(->> %
```

Comment:
This is only used in one spot below, so you should inline the code instead of making this a function.